lib/mailgun/events/events.rb

```diff
@@ -87,12 +87,22 @@ def get_events(params = nil, paging = nil)
     #
     # Return is irrelevant.
     def extract_paging(response)
-      # This is pretty hackish. But the URL will never change in API v2.
-      @paging_next = response.to_h['paging']['next'].split('/')[6]
-      @paging_previous = response.to_h['paging']['previous'].split('/')[6]
-    rescue
-      @paging_next = nil
-      @paging_previous = nil
+      next_page_url = response.to_h.dig('paging', 'next')
```

Comment:
@pirogoeth
Thanks for the swift feedback!
That really a good point. Let me work on it as per your suggestion.

(I tested it with my local env, and totally forgot `dig` is relatively a new feature...)